com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -875,6 +875,23 @@ protected virtual void Update()
 
             m_LocalAuthoritativeNetworkState.IsTeleportingNextFrame = false;
         }
+        // todo this is currently in update, to be able to catch any transform changes. A FixedUpdate mode could be added to be less intense, but it'd be
+        // conditional to users only making transform update changes in FixedUpdate.
+        protected virtual void Update()
```

Comment:
I can understand the idea and it makes sense to me, as in what you are trying to achieve. However, I would never want to have all my hundreds of NetworkTransform objects to check each frame or even multiple times per frame whether a boolean is set to false or true.

We can improve this by having a Manager that would run the updates for them and the NetworkTransform would subscribe on Start to either FixedUpdate/Update delegates.

Reply:
Yeah...I already knew this portion of the update would kick off a big debate... 
So, I might bring it up next week in the weekly discussion or create an earlier meeting to discuss this one particular change.   The really short version is that there isn't just a simple fix in the integration test itself for the issue that was happening on stand alone test runner builds as it has to do with the "unrealistic" timing issues that happen due to MultiInstanceHelpers, BaseMultiInstanceTest, SIPTransport, and how NetworkTransform initializes itself.
But yeah...I knew there was going to be a bunch of discussion around this fix... :)

Reply:
@Cosmin-B 
> I can understand the idea and it makes sense to me, as in what you are trying to achieve. However, I would never want to have all my hundreds of NetworkTransform objects to check each frame or even multiple times per frame whether a boolean is set to false or true.

I get your point, but the cost of a boolean test is probably much lower than the call to update (or fixedUpdate) itself from the engine. Once you put the object in the engine, the cost to update it has already been incurred, no?

Reply:
I don't consider it lower, almost at all, in a case where you have NetworkUpdateManager (which is a Unity best practice) you only pay the cost of one FixedUpdate/Update call from the engine because the rest of the calls to its subscribers would be similar to calling a C# function as you would do normally, which means that the cost of the Unity Events would not be incurred as they simply won't be there.

That is how I would be implementing these "updates" myself.

Reply:
There is a new approach that I just pushed which does not involve making adjustments to NetworkTransform.  
Still need to verify it works on consoles, but hopefully the adjustment will work and then it is just a matter of approving the changes to the test itself.